pkg/skuba/actions/cluster/init/init_test.go

```diff
@@ -142,6 +144,68 @@ func TestInitCloudProvider(t *testing.T) {
 	}
 }
 
+func TestInitCiliumCniPlugin(t *testing.T) {
+	clusterName := "testCluster"
+	controlPlane := "http://k8s.example.com"
+	k8sDesiredVersion := ""
+	strictCapDefaults := true
+	ciliumCni := "cilium"
+	initConf, err := NewInitConfiguration(
+		clusterName,
+		"",
+		controlPlane,
+		k8sDesiredVersion,
+		strictCapDefaults,
+		ciliumCni)
+
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	ctx, err := switchToTemporaryDirectory()
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if err = Init(initConf); err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	defer switchBackAndCleanFilesystem(ctx)
+	ciliumManifestPath := "addons/" + ciliumCni + "/base/" + ciliumCni + ".yaml"
+	found, err := doesFileExist(ctx, clusterName, ciliumManifestPath)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if !found {
+		t.Error("Cilium manifests not properly rendered")
```

Comment:
If an error happens, it should return. Otherwise, the following code would run.

Reply:
Done